koku/masu/processor/ocp/ocp_report_db_cleaner.py

```diff
@@ -118,17 +119,16 @@ def purge_expired_report_data_by_date(self, expired_date, simulate=False):
                 all_cluster_ids.add(usage_period.cluster_id)
                 all_period_starts.add(str(usage_period.report_period_start))
 
-            LOG.info(
-                log_json(
-                    msg="removing all data related to cluster_ids",
-                    report_periods=all_report_periods,
-                    cluster_ids=all_cluster_ids,
-                    period_starts=all_period_starts,
-                    schema=self._schema,
-                )
-            )
-
             if not simulate:
+                LOG.info(
+                    log_json(
+                        msg="removing all data related to cluster_ids",
+                        report_periods=all_report_periods,
+                        cluster_ids=all_cluster_ids,
+                        period_starts=all_period_starts,
```

Comment:
Are we doing anything with these?

Reply:
Nothing but unuseful logging. I removed them.